scripts/package-harvester-os

```diff
@@ -124,6 +124,13 @@ xorriso -volid "$(yq '.iso.label' manifest.yaml)" \
 # Cleanup
 rm -rf "${extract_dir}"
 
+if [ "${BUILD_QCOW}" == "true" ]; then
+  echo "generating harvester install mode qcow"
+  qemu-img create -f raw -o size=250G ${ARTIFACTS_DIR}/${PROJECT_PREFIX}-amd64.raw
```

Comment:
In the end, only raw format disks are generated, no qcow2 disk, is this as expected?

Reply:
yes we only expect a raw image

Reply:
my concerns:
1. raw takes up a lot of space
2. It does not match the BUILD_QCOW variable name and the `generating harvester install mode qcow` message and the PR description

Reply:
i can change those.
we need a raw, as I am not sure we can `dd` a qcow2 image to a physical disk

Reply:
can we convert the raw image in dist/artifacts to qcow2?

Reply:
We can keep both raw and qcow2

Reply:
yeah we can keep both. I will make the changes. unfortunately due to the persistent path sizing requirements this has now grown to a minimum of 250G. Pre 1.2.0-rc this used to be 60G.